README.md

```diff
@@ -1,19 +1,9 @@
 <img src="https://opensearch.org/assets/img/opensearch-logo-themed.svg" height="64px">
 
-[![Chat](https://img.shields.io/badge/chat-on%20forums-blue)](https://forum.opensearch.org/c/opensearch/)
-[![Documentation](https://img.shields.io/badge/documentation-reference-blue)](https://opensearch.org/docs/latest/opensearch/index/)
```

Comment:
&lt;subjective&gt;
Keeping chat / docs on this page seem useful.  Maybe if we could see if we are getting referrals or not via these links we could make a better call to remove them.
&lt;/subjective&gt;

Reply:
My rationale here is that these links are already available on this page in the "Project Resources" section and that seems like a more logical place than a badge at the top.